server/server.py

```diff
@@ -10,8 +10,13 @@
 DJANGO_USERS_URI = 'http://localhost:8000/users'
 DJANGO_GAME_URI = 'http://localhost:8000/games'
 
-users_connected = set()
 app = FastAPI()
+users_connected = set()
+
+
+@app.get("/")
```

Comment:
nit: some routes can go in a constants/routes file, which gives them more semantic meaning, like:

`URL_BASE = '/'`